packages/calcite-components/src/components/action-menu/action-menu.e2e.ts

```diff
@@ -287,6 +287,40 @@ describe("calcite-action-menu", () => {
     expect(await tooltipPositionContainer.isVisible()).toBe(false);
   });
 
+  it("should stop propagation of popover events", async () => {
```

Comment:
We might need a testing helper for this, but we can hold off until https://github.com/Esri/calcite-design-system/issues/12060 is fleshed out more.